app/assets/stylesheets/petitions/admin/_temporary_tab_nav.scss

```diff
@@ -0,0 +1,16 @@
+.admin {
```

Comment:
Why all the rules here wrapped in `.admin`, the child element seems to be referred with id and the `.admin` itself does not have any styles?